src/nupic/regions/SPRegion.py

```diff
@@ -159,29 +159,25 @@ def getConstraints(arg):
       description='Total number of columns (coincidences).',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     inputWidth=dict(
       description='Size of inputs to the SP.',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     spInputNonZeros=dict(
       description='The indices of the non-zero inputs to the spatial pooler',
       accessMode='Read',
       dataType='UInt32',
-      count=0,
-      constraints=''),
```

Comment:
@go-bears, what is the reason for removing the "constraints" property? It's not mentioned in issue #2398.